qan-api2/models/reporter.go

```diff
@@ -510,8 +511,17 @@ func (r *Reporter) SelectFilters(ctx context.Context, periodStartFromSec, period
 			if mainMetricPerSec == 0 {
 				total = totals[label.key]
 			}
+
+			var value string
+			switch v := label.value.(type) {
+			case string:
+				value = v
+			case []string:
+				value = strings.Join(v, ",")
```

Comment:
Is it worth adding a `default` here and at least log we received unexpected value?

Reply:
For unexpected values value will be empty anyway, but sure why not.

Reply:
@michal-kralik I took a look and its inside of loop. We dont have logger here passed and it can return only error. I am suggesting to keep it as it is to dont break whole loop and everything else in case of problem. Now we just skipping it.